osquery/core/system.cpp

```diff
@@ -215,16 +215,18 @@ std::string getHostIdentifier() {
       result = getEphemeralUUID(ident);
     } else if (FLAGS_host_identifier == "specified") {
       result = getSpecifiedUUID(ident);
-    } else {
+    }
+
+    if (!result.ok()) {
+      // TODO log result.getMessage() when getCode() == 1 once logging
```

Comment:
Instead of a todo can you include a 1-line note with a Github issue #?


Just a small nit, otherwise good to go!